docs: clarify use of `on delete cascade` for managing user data (#10963)
supabase/supabase: apps/docs/pages/guides/auth/managing-user-data.mdx

alter table public.profiles enable row level security;
```

<Admonition type="note">

> Make sure you specify the `on delete cascade` clause whenever referencing `auth.users`. Not including it may cause problems when deleting users.

Reviewer: ```suggestion
Make sure to specify the `on delete cascade` clause when referencing `auth.users`. Omitting it may cause problems when deleting users.
```
